### DISCUSSION TITLE
Alist 挂载阿里云成功，但无法成功读取阿里云盘内容

### DISCUSSION THREAD

**Discussion by george6066** [AlistGo/alist#5077] [Q&A]
完全无法显示！！！如图：
<img width="1280" alt="image" src="https://github.com/alist-org/alist/assets/142827435/9c26c2b6-d3b5-4487-97ea-a574a2fedf0e">
<img width="1280" alt="image" src="https://github.com/alist-org/alist/assets/142827435/1659511c-2b3d-4fd3-ba87-16ef35ef920b">

**Comment by Troray** [comment]
文档看过了吗？[https://alist.nn.ci/zh/faq/why.html](https://alist.nn.ci/zh/faq/why.html)

**Comment by anwen-anyi** [comment]
截图你的AList 阿里云盘open 配置页面

**Comment by xhofe** [comment]
https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E4%BA%91%E7%9B%98%E7%B1%BB%E5%9E%8B
